yaml-tests/src/test/resources/transactions-tests.yamsql

```diff
@@ -37,21 +37,29 @@
 #       whatever statements they want in whichever location.
 ---
 options:
-    supported_version: 4.3.5.0
+    supported_version: !current_version
 ---
 schema_template:
-    create table t1(id bigint, col1 bigint, primary key(id))
+    create table table_t1(id bigint, col1 bigint, primary key(id))
+---
+transaction_setups:
```

Comment:
I looked into using yaml's repeated nodes feature, but since each test_block is a separate "document" you can't really do that, so instead I'm adding our own reference section.